OpenRA.Game/Traits/SelectionDecorations.cs

```diff
@@ -97,8 +97,9 @@ IEnumerable<IRenderable> DrawPips(WorldRenderer wr, Actor self, int2 basePositio
 				{
 					if (pipxyOffset.X + pipSize.X >= width)
 					{
-						pipxyOffset.X = 0;
-						pipxyOffset.Y -= pipSize.Y;
+						pipxyOffset = pipxyOffset
```

Comment:
I don't think it is necessary to chain these methods here instead of directly creating a `new int2()`. Instead you are creating 2 and only ending up with 1.